stubs/flake8-plugin-utils/flake8_plugin_utils/plugin.pyi

```diff
@@ -18,7 +18,10 @@ class Error:
 
 class Visitor(ast.NodeVisitor, Generic[TConfig]):
     errors: list[Error]
-    def __init__(self, config: TConfig | None = ...) -> None: ...
+    @overload
+    def __init__(self, config: TConfig) -> None: ...
+    @overload
+    def __init__(self, config: None = ...) -> None: ...
```

Comment:
These overloads should be swapped because otherwise `None` will be bound to  `TConfig`.

Reply:
Ahh, good spot!